web/portal/brand/src/entities/RetailAccount/RetailAccount.tsx

```diff
@@ -8,10 +8,12 @@ import { foreignKeyGetter } from './ForeignKeyGetter';
 import { RetailAccountProperties } from './RetailAccountProperties';
 import foreignKeyResolver from './ForeignKeyResolver';
 import StatusIcon from './Field/StatusIcon';
+import Password from '../ResidentialDevice/Field/Password';
```

Comment:
Is ok to reushe this field? Or should RetailAccount have its own Password.

Reply:
I'd rather reuse it than copy it but it's debatable